itest/rust/src/register_tests/option_ffi_test.rs

```diff
@@ -97,7 +97,7 @@ impl OptionFfiTest {
 }
 
 #[derive(GodotClass)]
-#[class(init)]
+#[class(init, base=Node)]
```

Comment:
Is this necessary? We now have to explicitly clean up in `ManualFfiTests.gd` -- a `Resource` would not work here?

Reply:
More than sufficient for given test, I changed base to RefCounted & optional_export to Resource